### PR TITLE
LPS-199337 Turn interactive the paths on Web Content

### PR DESCRIPTION
**Pull request by BarbaraCabrera**
## Motivation

[Link to ticket](https://liferay.atlassian.net/browse/LPS-199337)

## Test scenario 1

1. Create a Some Web Contents and WC folders.
2. Use the searching bar to search them and check the path.

https://github.com/liferay-echo/liferay-portal/assets/93203423/6e788c09-e3a8-4e8e-be1e-39dbca431116

### CODE REVIEW

**Comment by liferay-echo** [comment]
Thank you for submitting this PR. 👏

Make sure it follows the [Echo Pull Request Template](https://gist.github.com/liferay-echo/5a4455285ff1800f9da146816d85e67d#file-pull-request-template-md) if you haven't already done so. 

Learn more about it [here](https://liferay.atlassian.net/wiki/spaces/ENGECHO/pages/2027455014/Echo+Pull+Request+Etiquette).

File: modules/apps/journal/journal-web/src/main/resources/META-INF/resources/view_entries.jsp [hunk 1]
```diff
@@ -121,10 +121,19 @@ Map<String, Object> componentContext = journalDisplayContext.getComponentContext
 								<liferay-ui:message arguments="<%= new String[] {modifiedDateDescription, HtmlUtil.escape(curArticle.getStatusByUserName())} %>" key="modified-x-ago-by-x" />
 							</span>
 
+							<%
+							PortletURL articleURL = PortletURLBuilder.createRenderURL(
+								liferayPortletResponse
+							).setParameter(
+								"folderId", curArticle.getFolderId()
+							).buildPortletURL();
```
Comment (ealonso) [inline modules/apps/journal/journal-web/src/main/resources/META-INF/resources/view_entries.jsp:129]: You can use .buildString(), in that way you don't need toString

File: modules/apps/journal/journal-web/src/main/resources/META-INF/resources/view_entries.jsp [hunk 2]
```diff
@@ -121,10 +121,19 @@ Map<String, Object> componentContext = journalDisplayContext.getComponentContext
...
+							%>
+
 							<c:if test="<%= journalDisplayContext.isSearch() && ((curArticle.getFolderId() <= 0) || JournalFolderPermission.contains(permissionChecker, curArticle.getFolder(), ActionKeys.VIEW)) %>">
-								<h5>
-									<%= journalDisplayContext.getAbsolutePath(curArticle.getFolderId()) %>
-								</h5>
+								<clay:link
+									href="<%= articleURL.toString() %>"
+									label="<%= journalDisplayContext.getAbsolutePath(curArticle.getFolderId()) %>"
```
Comment (ealonso) [inline modules/apps/journal/journal-web/src/main/resources/META-INF/resources/view_entries.jsp:135]: Why we are doing it all path link and not each element in the path?

Comment (BarbaraCabrera) [reply modules/apps/journal/journal-web/src/main/resources/META-INF/resources/view_entries.jsp:135]: We decide this after discussing it with design and pm. It was decided that for now we would leave the path with a link to the folder and that a ticket would be opened to have the clay breadcrumb taglib as the one we currently use does not work as we want for these cases.

**Comment by BarbaraCabrera** [comment]
ci:test:sf

**Comment by ealonso** [comment]
see : https://github.com/BarbaraCabrera/liferay-portal/pull/27

**Pull request closed without merging**